api/lib/config.js

```diff
@@ -73,7 +73,7 @@ module.exports = (function () {
     logging: {
       enabled: isFeatureEnabled(process.env.LOG_ENABLED),
       logLevel: process.env.LOG_LEVEL || 'info',
-      logForHumans: isFeatureEnabled(process.env.LOG_FOR_HUMANS),
+      logForHumans: process.stdout.isTTY && !(process.env.LOG_FOR_HUMANS === 'false'),
```

Comment:
Hypothèse de Nicolas: on veut le prettry-print par défaut si c'est un terminal SAUF si on veut tester le logging 
Ca donne (j'ai poussé un commit)
```suggestion
      logForHumans: process.stdout.isTTY && !(process.env.LOG_FOR_HUMANS === 'false'),
```